poetry/orb.yaml

```diff
@@ -66,6 +66,11 @@ jobs:
           Name of the repository you wish to upload to. If using a non-default
           repository, you'll need to define it in your pyproject.toml.
         type: string
+      repository-url:
+        description: >
+          URL of repository not provided in pyproject.toml. Uses repository
+          parameter as name of repository for configuration.
```

Comment:
Any restrictions around whether `repository` needs to exist/not-exist in pyproject.toml when this is set? eg. `repository`'s description above says `If using a non-default repository, you'll need to define it in your pyproject.toml.`. Is that still the case even if repository-url is set?

Reply:
@TheKevJames,

Good questions.

The repository does not need to exist in `pyproject.toml`.

If `repository-url` is set, poetry will correctly publish the files whether the repository is also declared in the `pyproject.toml`.

I've tested this by manually updating `~/.config/pypoetry` and by running `poetry config repositories.foo https://example.com/simple` (the latter just updates `~/.config/pypoetry`). And both correctly configure the repository to read and write from.

Reply:
Cool -- then yeah, we should probably update either this description or `respository`'s: `If using a non-default repository, you'll need to define it in your pyproject.toml.` is no longer correct in some cases.

Reply:
I see what you are saying. I've updated the descriptions accordingly.